### PR TITLE
Clarify that action is related to background scan only

### LINKED ISSUE #129: "Log only" acts just the same as "Delete"
Hi,

I installed files_antivirus today mostly successfully, however I find that the option "Log only" is not working . With option "Log only", file gets blocked with a message "Virus Eicar-Test-Signature is detected in the file. Upload cannot be completed." so does option "Delete". 

Shouldn't they act differently? I expect that "Log only" gives a warning message, a log info and maybe a confirm button, but not prevent files from uploading.
## My system info is:

Operating System: CentOS 7.1
Web server: Apache 
Database: MySQL
PHP version: 7
ownCloud version: 9.1
Updated from an older ownCloud or fresh install: updated from 8
Antivirus App for files 0.9.0.0 with patch #127 

Thank you very much!

### PR DESCRIPTION
**Pull request by VicDeo**
Closes #129

### CODE REVIEW

**Comment by VicDeo** [comment]
@PVince81 please proof-read

File: templates/settings.php [hunk 1]
```diff
@@ -34,7 +34,7 @@
 				/>
 				<label for="av_max_file_size" class="a-left"><?php p($l->t('bytes'))?></label>
 			</p>
-			<p class="infected_action"><label for="av_infected_action"><?php p($l->t('Action for infected files found while scanning'));?></label>
+			<p class="infected_action"><label for="av_infected_action"><?php p($l->t('When infected files are found by cron job'));?></label>
```
Comment (PVince81) [inline templates/settings.php]: "When infected files were found during a background scan" ?

Or is it too long ?

Comment (VicDeo) [inline templates/settings.php]: @PVince81 fits into the setting section

**Comment by VicDeo** [comment]
@PVince81 updated

**Comment by PVince81** [comment]
:+1:

**Pull request merged**